### PR TITLE
Moved Aux from Music player to Drivers and changed subsequent CAN data

### PR DESCRIPTION
**Pull request by Reddragoneer1**
Moved Aux from Music player to Driver. Previous Music player had Can data in hexadecimal with 1, 2, 3, 4, 10. 3 should not be there because it would mean 2 outputs, Aux and Volume Up.

### CODE REVIEW

File: EpsilonCCS/Inc/Data/DriverControlData.h [hunk 1]
```diff
@@ -15,11 +15,10 @@
...
 // Music Inputs
-#define AUX_MASK 0x01
-#define VOLUME_UP_MASK 0x02
-#define VOLUME_DOWN_MASK 0x04
-#define NEXT_SONG_MASK 0x08
-#define PREV_SONG_MASK 0x10
+#define VOLUME_UP_MASK 0x01
+#define VOLUME_DOWN_MASK 0x02
+#define NEXT_SONG_MASK 0x04
+#define PREV_SONG_MASK 0x8
```
Comment (bill-luu) [inline EpsilonCCS/Inc/Data/DriverControlData.h]: Use `0x08` for consistency with the rest.

File: EpsilonDriverControls/Src/DriverControls.c [hunk 2]
```diff
@@ -121,7 +120,7 @@ void sendDriverTask(void const* arg)
         msg->Data[3] |= 0x08 * !HAL_GPIO_ReadPin(PUSH_TO_TALK_GPIO_Port, PUSH_TO_TALK_Pin);
         msg->Data[3] |= 0x10 * !HAL_GPIO_ReadPin(HORN_GPIO_Port, HORN_Pin);
         msg->Data[3] |= 0x20 * !HAL_GPIO_ReadPin(RESET_GPIO_Port, RESET_Pin);
-        // Send CAN Message
```
Comment (bill-luu) [inline EpsilonDriverControls/Src/DriverControls.c]: The comment that was here got removed, let's put it back.

File: EpsilonCCS/Inc/Data/DriverControlData.h (hunk 1 above)
Comment (Reddragoneer1) [reply EpsilonCCS/Inc/Data/DriverControlData.h]: Oh, oops

**Comment by bill-luu** [comment]
Nice job!

In the future, add "Fixes #<issueNumber>" to the PR description to link it with the issue, so that when the PR is merged, the issue is closed as well!

In your case, this would close #34

**Pull request merged**